abci/client/grpc_client.go

```diff
@@ -33,6 +30,11 @@ type grpcClient struct {
 	resCb func(*types.Request, *types.Response) // listens to all callbacks
 }
 
+var _ Client = (*grpcClient)(nil)
+
+// NewGRPCClient creates a gRPC client, which will connect to addr upon the
+// start. Note Client#Start returns an error if connection is unsucessful and
+// mustConnect is true.
 func NewGRPCClient(addr string, mustConnect bool) Client {
```

Comment:
It doesn't look like we're actually using the context for the gRPC requests anywhere.